midi_app_controller/config_files/controllers/x_touch_mini.yaml

```diff
@@ -4,6 +4,8 @@ button_value_on: 127
 knob_value_min: 0
 knob_value_max: 127
 default_channel: 11
+preferred_midi_in: "Midi Through:Midi Through Port-0 14:0"
+preferred_midi_out: "Midi Through:Midi Through Port-0 14:0"
```

Comment:
Incorrect port names. I would also change them in `docs/controllers.md` because someone may think that these ports will work with all controllers.